foreshadow/utils/common.py

```diff
@@ -73,10 +73,9 @@ def get_transformer(class_name, source_lib=None):
         sources = OrderedDict(
             (source, import_module(source))
             for source in [
-                "foreshadow.transformers.concrete",
-                "foreshadow.transformers.smart",
-                "foreshadow.core",
-                "foreshadow.cleaners.internals",
+                "foreshadow.concrete.internals",
```

Comment:
Can we just check concrete instead of both internals and externals?

Reply:
Yeah.